### DISCUSSION TITLE
HydroInspector Access

### DISCUSSION THREAD

**Discussion by zhong-industries** [NCAR/wrf_hydro_nwm_public#762] [General]
Is there any way for a researcher such as myself to access the the HydroInspector tool for WRF Hydro mentioned here: https://ral.ucar.edu/solutions/products/hydroinspector

I did find a link at https://hydro.rap.ucar.edu/HydroInspector, but it appears to require authentication

**Comment by scrasmussen** [comment]
Hi, the HydroInspector is currently not publicly available and open-source at the time.
